### PR TITLE
Creating caterpillar.md

### PR DESCRIPTION
**Pull request by markymauro13**
# Description

Added the company Caterpillar to the "companies" folder by writing the caterpillar.md file.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Company data addition
- [ ] Bug fix
- [ ] New feature

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

All I did was add caterpillar.md to the project under the folder, "companies"

**Test Configuration**:
* Browser:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: companies/caterpillar.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+fortune_rank: 62
+name: Caterpillar
+tags: ["fortune500", "construction", "machinery", "tech", "insurance", "energy"]
+image: name-in-images-folder.png
```
Comment (kelseyhuse30) [inline companies/caterpillar.md:5]:
```suggestion
image: catepillar.png
```

Look like this is what your file is named :) (I haven't added good instructions on this piece of the file yet, still working on that)

**Review by kelseyhuse30 — CHANGES_REQUESTED** [review]
This is looking great @markymauro13 ! Thank you for taking the initiative to find this information and add it to the site.  Just a few quick fixes to make and then it'll be ready to deploy ⭐

**Comment by markymauro13** [comment]
Hey Kelsey! I think it should be fixed now!

File: companies/caterpillar.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+fortune_rank: 62
+name: Caterpillar
+tags: ["fortune500", "construction", "machinery", "tech", "insurance", "energy"]
+image: caterpillar.jpg
+blm_statements:
+  - url: https://www.caterpillar.com/en/company/the-fight-against-racism-and-injustice.html
+    date_posted: 2020-06-02
+    type: Blog
+    raw_text: Chairman and CEO Jim Umpleby shared the following message with employees,
```
Comment (kelseyhuse30) [inline companies/caterpillar.md:10]: It looks like it's still erroring :/ Try this and make sure the entire block of raw_text is indented once more :)

```suggestion
    raw_text: >
       Chairman and CEO Jim Umpleby shared the following message with employees,
```

File: companies/caterpillar.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+fortune_rank: 62
+name: Caterpillar
+tags: ["fortune500", "construction", "machinery", "tech", "insurance", "energy"]
+image: caterpillar.jpg
```
Comment (kelseyhuse30) [inline companies/caterpillar.md:5]: Looks great here 👍🏻 👍🏻

**Pull request merged**